test/NfRel/NonSimple.jl

```diff
@@ -0,0 +1,20 @@
+@testset "NfRelNS" begin
+  Qx, x = FlintQQ["x"]
+  K, _ = number_field(x^2 - 2)
+  Ky, y = K["y"]
+  L, (a, b) = @inferred number_field([x^2 - 3, x^3 - 5])
```

Comment:
Do you want to use `y` here? At the moment this is not calling the new methods.

Reply:
yup, thanks